gateway-discovery-cm/src/main/java/org/apache/knox/gateway/topology/discovery/cm/model/phoenix/PhoenixServiceModelGenerator.java

```diff
@@ -58,14 +59,24 @@ public ServiceModel generateService(ApiService       service,
                                       ApiServiceConfig serviceConfig,
                                       ApiRole          role,
                                       ApiConfigList    roleConfig) {
+
     String hostname = role.getHostRef().getHostname();
-    // Phoenix Query Server does not support https
+    String sslEnabledRaw = getRoleConfigValue(roleConfig, SSL_ENABLED);
```

Comment:
This can be simplified. No need to store `sslEnabledRaw`. just do something like:

```
if (Boolean.parseBoolean(getRoleConfigValue(roleConfig, SSL_ENABLED))) {
    scheme = "https";
}
```

`Boolean.parseBoolean` will handle null correctly and return false.